app/components/clinic/RpmReportConfigForm.js

```diff
@@ -157,33 +154,85 @@ export const RpmReportConfigForm = props => {
         [localConfigKey]: pick(values, ['timezone']),
       });
     },
-    validationSchema,
+    validationSchema: rpmReportConfigSchema(utcDayShift),
   });
 
   const {
     errors,
     setFieldValue,
     values,
     setValues,
-    validateForm
+    validateForm,
   } = formikContext;
 
   // Set to default state when dialog is newly opened
   useEffect(() => {
     if (open) {
+      // Set global moment to use UTC timezone when config modal opens. This is required for our
+      // use within this form, since React-dates (used for our datepicker components) does not allow
+      // overriding the default moment-provided timezone, and we need to work with UTC dates for
+      // consistent behaviour across various timezones
+      setMomentToUTC();
```

Comment:
Minor nit: this has the feel of a use-before-define to me since these utility functions are defined after the `useEffect` sets.

Reply:
Because the functions are declared as functions, they do get hoisted above where they're called, but I'm fine with shifting their definitions upward to read better.

Reply:
Done